umbra-js/src/classes/Umbra.ts

```diff
@@ -8,7 +8,6 @@ import {
   BigNumber,
```

Comment:
(_random placement_)

CI coverage is now super low for `Umbra.ts`, not sure why?

Reply:
Hmm let's see, here's what I get on master:
<img width="1556" alt="image" src="https://user-images.githubusercontent.com/17163988/230154724-931b7e7e-8953-4d95-a519-6b86f931ffd2.png">
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/17163988/230154765-c5e71e97-8327-4594-b04a-dbcb09fc2d27.png">


And on this branch:
<img width="1553" alt="image" src="https://user-images.githubusercontent.com/17163988/230154043-7800a610-6781-433e-af24-74b5b8692505.png">
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/17163988/230154168-2a146398-4bc9-4422-b05b-9f1173f111bd.png">

That's odd, it seems coverage went down for things that weren't even changed on this PR? 🤨

Reply:
This is now resolved, the issue was a `.only` was accidentally committed which was removed in https://github.com/ScopeLift/umbra-protocol/pull/426/commits/93f3b34677c2160030cc414a5405d6d1a201f0a4